wgsl/index.bs

```diff
@@ -2890,10 +2890,10 @@ The originating variable for a formal parameter of a function depends on the
 [=call site|call sites=] for the function.
 Different call sites may supply pointers into different originating variables.
 
-If a reference or pointer access is out of bounds, an <dfn noexport>invalid
-memory reference</dfn> is produced.
+If a reference or pointer access is out of bounds, an
+<dfn noexport>invalid memory reference</dfn> is produced.
 [=Load Rule|Loads=] from an invalid reference return one of:
-    * a value from any [=memory locations|memory location(s)=] of the [[WebGPU#buffers|WebGPU buffer]]
+    * a value from any [=memory locations|memory location(s)=]
```

Comment:
This is insufficient. I get what is being attempted, but to satisfy Vulkan's robust buffer access you have to allow more than just the locations of the variable. The variable might be declared as 128B, but if the buffer bound is 256B you can get values from bytes beyond the 128th.

So maybe the answer is it has to be broken down by address space. Uniform and storage have the behaviour listed, while private, function and workgroup the behaviour your PR suggests.